docs/user/openstack/bind-vfio-pci-kernel-module.md

```diff
@@ -0,0 +1,144 @@
+# Binding the vfio-pci kernel module to vhostuser port.
+
+When a VM is attached to a port of type dpdkvhostuserclient, as is the case when vms are
+spun up inside an OpenStack deployment configured for [ovs-dpdk][1] it is possible to use a pmd driver like the [VFIO
+driver][1] on the port.
+Before the interface can be used by a pmd driver,  the vfio-pci kernel module needs to be loaded and bound to
+the desired interface of the vm.
+
+In a regular VM running linux OS, the cli commands would look something like:
```

Comment:
`s/linux/Linux/` and `s/cli/CLI/`. Also, this is me being OCDish, but can we have newlines before and after code fences, please? :innocent: